toolkit/scripts/sodiff/mariner-sodiff.sh

```diff
@@ -2,11 +2,36 @@
 
 # Required binaries:
 # rpm and dnf
+sodiff_script_error=false
```

Comment:
Might be worth to also add `set -eo pipefail` to always error out on unexpected issues.

Reply:
Good call, Added

Reply:
scratch that, adding `set -e` is causing the script to fail due to grep failures on 54 becoming hard errors, which are normally suppressed.